staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/validation/validation.go

```diff
@@ -39,7 +39,7 @@ func NewSchemaValidator(customResourceValidation *apiextensions.CustomResourceVa
 			return nil, nil, err
 		}
 	}
-	return validate.NewSchemaValidator(openapiSchema, nil, "", strfmt.Default, validate.DisableObjectArrayTypeCheck(true)), openapiSchema, nil
```

Comment:
I'm confused... I thought we wanted this option ("we forgot to disable go-openapi meta checks for default validation.")